code/game/gamemodes/cult/blood_magic.dm

```diff
@@ -644,6 +647,47 @@
 		C.put_in_hands(new /obj/item/restraints/legcuffs/bola/cult(user))
 		C.visible_message("<span class='warning'>Otherworldly [armour ? "armour" : "equipment"] suddenly appears on [C]!</span>")
 		..()
+//Used by blood rite, to recharge things like viel shifter or the cultest shielded robes
+/obj/item/melee/blood_magic/empower
+	name = "Blood Recharge"
+	desc = "Can be used on some cult items, to restore them to their previous state."
+	invocation = "Ditans Gut'ura Inpulsa!"
+	color = "#9c0651"
+	has_source = FALSE //special, only availible for a blood cost.
+
+/obj/item/melee/blood_magic/empower/afterattack(atom/target, mob/user, proximity_flag, click_parameters)
+	if(proximity_flag)
+
+		//Shielded suit
+		if(istype(target, /obj/item/clothing/suit/hooded/cultrobes/cult_shield))
+			var/obj/item/clothing/suit/hooded/cultrobes/cult_shield/C = target
+			if(C.current_charges < 3)
+				uses--
+				to_chat(user, "<span class='warning'>You empower the [target.name] with blood, recharging its shields!</span>")
```

Comment:
By putting `[target]`, byond will automatically write "The shielded robes" or whatever the item name may be.
```suggestion
				to_chat(user, "<span class='warning'>You empower [target] with blood, recharging its shields!</span>")
```